### PR TITLE
PMM 7259: link to  activate IA

### PR DESCRIPTION
**Pull request by fabio-silva**
What this PR does / why we need it: When reaching IA page and IA is disabled, there will be a panel with a warning and a link to PMM settings, so that users can enable it right away.

Which issue(s) this PR fixes: https://jira.percona.com/browse/PMM-7259

FB https://github.com/Percona-Lab/pmm-submodules/pull/1454

# Implementation details

## `<EmptyBlock>`
This component just encapsulates the `class=empty` blocks we have on IA, letting children being rendered within.
Also, `dataQa` prop allows the `data-qa` attribute to be passed to the wrapper.

## `<IntegratedAlerting>`
- After mounted, the page will call `/v1/Settings/Get` and extract `alerting_enabled` from there. Based on that, either the regular IA tabs are shown, or just an empty block with a message like shown in the screenshot below.

## Screenshots
Screenshot 2021-02-01 at 21 55 26

### CODE REVIEW

File: public/app/features/integrated-alerting/IntegratedAlertingPage.test.tsx [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+import { IntegratedAlertingService } from './IntegratedAlerting.service';
+import { dataQa } from '@percona/platform-core';
+
+jest.mock('./IntegratedAlerting.service');
+jest.mock('./components/Alerts/Alerts.service');
 
 describe('IntegratedAlertingPage', () => {
-  it('should render component correctly', () => {
-    const wrapper = mount(<IntegratedAlertingPage />);
+  let wrapper: any;
```
Comment (tiagomotasantos) [inline public/app/features/integrated-alerting/IntegratedAlertingPage.test.tsx:13]:
```suggestion
  let wrapper: ReactWrapper;
```

File: public/app/features/integrated-alerting/IntegratedAlertingPage.tsx [hunk 2]
```diff
@@ -47,6 +50,62 @@ const IntegratedAlertingPage: FC = () => {
         ))}
       </TabsBar>
       <TabContent>{tabs.find(tab => tab.key === activeTab).component}</TabContent>
+    </>
+  );
+};
+
+export const IntegratedAlertingContent: FC<{ loadingSettings: boolean; alertingEnabled: boolean }> = ({
```
Comment (tiagomotasantos) [inline public/app/features/integrated-alerting/IntegratedAlertingPage.tsx:57]: We should keep one component per file, this three components can be separated. Also this one should have an interface for the props.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: public/app/features/integrated-alerting/components/EmptyBlock/EmptyBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { useStyles } from '@grafana/ui';
+import React from 'react';
+import { FC } from 'react';
+import { getStyles } from './EmptyBlock.styles';
```
Comment (tiagomotasantos) [inline public/app/features/integrated-alerting/components/EmptyBlock/EmptyBlock.tsx:4]: Please create an interface and put it in a file for the types.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: public/app/features/integrated-alerting/components/EmptyBlock/EmptyBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+export const EmptyBlock: FC<{ dataQa?: string }> = ({ children, dataQa }) => {
+  const style = useStyles(getStyles);
```
Comment (tiagomotasantos) [inline public/app/features/integrated-alerting/components/EmptyBlock/EmptyBlock.tsx:7]: Please create an interface and put it in a file for the types.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: public/app/features/integrated-alerting/IntegratedAlertingPage.test.tsx (hunk 1 above)
Comment (fabio-silva) [reply public/app/features/integrated-alerting/IntegratedAlertingPage.test.tsx]: 👍

File: public/app/features/integrated-alerting/IntegratedAlertingPage.tsx (hunk 2 above)
Comment (fabio-silva) [reply public/app/features/integrated-alerting/IntegratedAlertingPage.tsx]: Done 👍

File: public/app/features/integrated-alerting/components/EmptyBlock/EmptyBlock.tsx (hunk 4 above)
Comment (fabio-silva) [reply public/app/features/integrated-alerting/components/EmptyBlock/EmptyBlock.tsx:8]: Got it

**Review by tiagomotasantos — DISMISSED** [review]

**Review by nicolalamacchia — DISMISSED** [review]

**Comment by fabio-silva** [comment]
(New approval needed after merging master)

**Review by nicolalamacchia — APPROVED** [review]

**Pull request merged**